packages/react-app/components/home/RecentBuildsSection.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import NextLink from "next/link";
+import Address from "../Address";
+import Card from "../Card";
+import { Flex, Button, VStack, Container, Divider, Heading, chakra, Text, Link } from "@chakra-ui/react";
+import useCustomColorModes from "../../hooks/useCustomColorModes";
+import { formatDateFromNow } from "../../helpers/formatDateFromNow";
+
+const MAX_DESC_LEN = 150;
+const RecentBuildCard = ({ build }) => {
+  const { baseColor, baseBlue2Color, blueColor } = useCustomColorModes();
+  return (
+    <VStack key={build.id} align="start" spacing={5}>
+      <Card maxW="xs" display="flex" flexDirection="column" key={build}>
+        <VStack align="start" spacing={3} p={5} bg={baseColor}>
+          <chakra.span fontSize="sm">{formatDateFromNow(build.submittedTimestamp)}</chakra.span>
+          <Heading size="md" fontWeight="500">
+            {build.name}
+          </Heading>
+        </VStack>
+        <Divider />
+        <VStack align="start" spacing={3} p={5} bg={baseBlue2Color}>
+          <Text fontSize="sm">
+            {build.desc.length > MAX_DESC_LEN ? `${build.desc.substring(0, MAX_DESC_LEN)}...` : build.desc}
+          </Text>
+          <NextLink href={`/build/${build.id}`} passHref>
+            <Button
+              color="light.text"
+              fontWeight="500"
+              bg={blueColor}
+              px={4}
+              py={2}
+              _hover={{ opacity: 0.8, textDecoration: "underline" }}
+            >
+              Learn more
+            </Button>
```

Comment:
I created the "secondary" variant in this PR https://github.com/scaffold-eth/buidlguidl-v3/pull/99

Maybe we could use here too? (it might need to be improved, but at least it will be consistent!)

Reply:
Nice, now using secondary variant at [02c138d](https://github.com/scaffold-eth/buidlguidl-v3/pull/98/commits/02c138d138f8011b2960d02a716eb99ddf0afefa) 


I have removed the secondary variant hover background and just decrease opacity while hovering at https://github.com/scaffold-eth/buidlguidl-v3/pull/98/commits/097716d2abb5822b3be32c5e4ef77be90b38c24e  : 

Before it was getting too light : 

https://github.com/scaffold-eth/buidlguidl-v3/assets/80153681/ea380110-92f6-459e-92e9-685c16609aeb

After (with just opacity reduced) : 

https://github.com/scaffold-eth/buidlguidl-v3/assets/80153681/a584bf55-0c36-4a3d-8272-71065dd250f6

Lol not sure what's best or any better alternative

Reply:
I like the opacity! Thanks!